src/components/VWindow/VWindow.ts

```diff
@@ -0,0 +1,123 @@
+// Styles
+import '../../stylus/components/_windows.styl'
+
+// Mixins
+import Proxyable from '../../mixins/proxyable'
+
+// Utilities
+import {
+  addOnceEventListener,
+  convertToUnit
+} from '../../util/helpers'
+import mixins from '../../util/mixins'
+
+// Types
+import { VNode } from 'vue/types/vnode'
+
+export default mixins(
+  Proxyable
+  /* @vue/component */
+).extend({
+  name: 'v-window',
+
+  props: {
+    mode: String,
+    reverse: {
+      type: Boolean,
+      default: undefined
+    },
+    transition: String,
+    value: {
+      type: Number,
```

Comment:
Is there a reason for allowing only numbers? This component could be also used in date pickers (to DRY or to allow vertical scrolling) so it would be more natural to pass the string value instead of calculating numeric value of a date (for example as `10000 * year + 100 * month + day`). Also maybe in some other cases providing a numeric value might not be easy